cfme/infrastructure/virtual_machines.py

```diff
@@ -412,7 +353,9 @@ def cluster_id(self):
     class CfmeRelationship(object):
         relationship_form = Form(
             fields=[
-                ('server_select', Select("//*[@id='server_id']")),
+                ('server_select', {
+                    version.LOWEST: Select("//*[@id='server_id']"),
+                    "5.5": AngularSelect("server_id")}),
```

Comment:
**REQUIRED:** We don't support version lower than 5.6. So you can just leave AngularSelect without another one